### PR TITLE
[Doc Improvemet][metadata-keywords-changes][2333804]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/23b9be1a838149865de0292d05e701e8b045c775/msteams-platform%2Fwhats-new.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a8da48e5-be04-4ffc-89ba-d43df3ce364a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/get-started/glossary.md](https://github.com/MicrosoftDocs/msteams-docs/blob/72236d4ed404e0621e3911bd69f65850627aecba/msteams-platform%2Fget-started%2Fglossary.md) | 95 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dfa4c8a6-b3e7-4893-8d7d-2b787f7aa632) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/72236d4ed404e0621e3911bd69f65850627aecba/msteams-platform%2Fwhats-new.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/64b5ab6d-e7b6-42e2-b7f9-a8009d454a83) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/get-started/glossary.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 ---
 title: Microsoft Teams Developer Documentation - Glossary
-description: Glossary for Microsoft Teams Developer Documentation
+description: Learn about the glossary and terms used for Microsoft Teams Developer Documentation
```
Comment (ashishguptaiitb) [inline msteams-platform/get-started/glossary.md:3]: We aren't learning about glossary - just about the terms.

File: msteams-platform/whats-new.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
 ---
 title: What's new
-description: Describes all the new or latest developer features in Microsoft Teams
+description: Learn about all the new or latest developer features and improvements in Microsoft Teams
```
Comment (ashishguptaiitb) [inline msteams-platform/whats-new.md:3]: We aren't learning about improvements in Teams in this article. I'm editing this accordingly.
CC @surbhigupta12 @v-npaladugu

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/get-started/glossary.md](https://github.com/MicrosoftDocs/msteams-docs/blob/54a8eccdfdeac8a151e5dae3e58e23a702639c09/msteams-platform%2Fget-started%2Fglossary.md) | 95 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/76bbd726-efca-4a38-a4ac-46e7b5754d4f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/54a8eccdfdeac8a151e5dae3e58e23a702639c09/msteams-platform%2Fwhats-new.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9c0f25aa-3aca-4a5a-8295-48f34167470f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/get-started/get-started-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ae018bec9fd755df6d4458a1b806dbb2909ff0d9/msteams-platform%2Fget-started%2Fget-started-overview.md) | 97 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3b3fafe3-9826-47ad-9af6-5784a4d9356a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/get-started/glossary.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ae018bec9fd755df6d4458a1b806dbb2909ff0d9/msteams-platform%2Fget-started%2Fglossary.md) | 95 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ee7b668b-e316-4b03-b158-142eea4e086b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/overview-solution.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ae018bec9fd755df6d4458a1b806dbb2909ff0d9/msteams-platform%2Foverview-solution.md) | 98 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0bc61601-d9eb-4a08-9c0e-b62311478aa0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/overview-story.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ae018bec9fd755df6d4458a1b806dbb2909ff0d9/msteams-platform%2Foverview-story.md) | 86 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bdf5db65-53a8-4692-b90f-9f3c153df95a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ae018bec9fd755df6d4458a1b806dbb2909ff0d9/msteams-platform%2Fwhats-new.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/603695bf-18e9-4f91-aebe-384dc190821a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/get-started/get-started-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/41bfb46fc054af2a9f35548d7e645cf602b74173/msteams-platform%2Fget-started%2Fget-started-overview.md) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [msteams-platform/get-started/glossary.md](https://github.com/MicrosoftDocs/msteams-docs/blob/41bfb46fc054af2a9f35548d7e645cf602b74173/msteams-platform%2Fget-started%2Fglossary.md) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [msteams-platform/overview-explore.md](https://github.com/MicrosoftDocs/msteams-docs/blob/41bfb46fc054af2a9f35548d7e645cf602b74173/msteams-platform%2Foverview-explore.md) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [msteams-platform/overview-solution.md](https://github.com/MicrosoftDocs/msteams-docs/blob/41bfb46fc054af2a9f35548d7e645cf602b74173/msteams-platform%2Foverview-solution.md) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [msteams-platform/overview-story.md](https://github.com/MicrosoftDocs/msteams-docs/blob/41bfb46fc054af2a9f35548d7e645cf602b74173/msteams-platform%2Foverview-story.md) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [msteams-platform/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/41bfb46fc054af2a9f35548d7e645cf602b74173/msteams-platform%2Foverview.md) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/41bfb46fc054af2a9f35548d7e645cf602b74173/msteams-platform%2Fwhats-new.md) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/get-started/get-started-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8a1fbfb9c710cbe7736e2742b7f8da53d7703ca5/msteams-platform%2Fget-started%2Fget-started-overview.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d4d59384-8231-412d-893a-6d8c143edcf7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/get-started/glossary.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8a1fbfb9c710cbe7736e2742b7f8da53d7703ca5/msteams-platform%2Fget-started%2Fglossary.md) | 95 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e2bfdcd8-4184-41ce-8311-26146a2a83eb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/overview-explore.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8a1fbfb9c710cbe7736e2742b7f8da53d7703ca5/msteams-platform%2Foverview-explore.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/78e2c983-d24b-4056-998d-6bf3604f97d6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/overview-solution.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8a1fbfb9c710cbe7736e2742b7f8da53d7703ca5/msteams-platform%2Foverview-solution.md) | 98 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/05fe0924-8e06-42dc-8aa6-37955cad549a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/overview-story.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8a1fbfb9c710cbe7736e2742b7f8da53d7703ca5/msteams-platform%2Foverview-story.md) | 86 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9f247940-532b-4c6a-93f6-996fb0b62878) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8a1fbfb9c710cbe7736e2742b7f8da53d7703ca5/msteams-platform%2Foverview.md) | 98 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/db7cfce2-0504-443d-841f-5a2a252bea7c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8a1fbfb9c710cbe7736e2742b7f8da53d7703ca5/msteams-platform%2Fwhats-new.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a2919ad1-cbf1-49e5-9682-c377cf805155) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**